docker/Makefile

```diff
@@ -48,28 +49,39 @@ platformio/Dockerfile.log: platformio/Dockerfile | check-docker
 	| tee -a $@.part
 	mv $@.part $@
 
-.PHONY: flash
-flash:
+.PHONY: pio-flash pio-upload
+pio-flash pio-upload:
 	docker run --rm -ti \
-		--device /dev/${DEV} \
+		--device "${DEV}" \
 		--volume "$(shell pwd)/..":/mnt:rw \
 		-w /mnt \
 		--name flash \
 		lifesensor/platformio:$(PLATFORMIO_VERSION) \
 		/bin/sh -c \
-		"pio run -t clean; pio run -t upload; /bin/bash"
+		"pio run -t upload"
 
-.PHONY: monitor
-monitor:
+.PHONY: pio-monitor
+pio-monitor:
 	docker run --rm -ti \
-		--device /dev/${DEV} \
+		--device "${DEV}" \
 		--volume "$(shell pwd)/..":/mnt:rw \
 		--name monitor \
 		lifesensor/platformio:$(PLATFORMIO_VERSION) \
 		/bin/sh -c \
 		"python3 \
 		/home/developer/.platformio/packages/framework-espidf/tools/idf_monitor.py \
-		/mnt/.pio/build/pico32/firmware.elf"
+		/mnt/.pio/build/pico32/firmware.elf --port ${DEV}"
+
+.PHONY: pio-clean
+pio-clean:
```

Comment:
the container dependency and check if docker is installed is missing
`pio-flash pio-upload: | docker-platformio check-docker`